crates/red_knot_python_semantic/resources/mdtest/annotations/stdlib_typing_aliases.md

```diff
@@ -49,35 +65,54 @@ The aliases can be inherited from. Some of these are still partially or wholly T
 ```py
 import typing
 
-class A(typing.Dict): ...
+####################
+### Built-ins
+
+class _List(typing.List): ...
 
 # TODO: should have `Generic`, should not have `Unknown`
-reveal_type(A.__mro__)  # revealed: tuple[Literal[A], Literal[dict], Unknown, Literal[object]]
+reveal_type(_List.__mro__)  # revealed: tuple[Literal[_List], Literal[list], Unknown, Literal[object]]
 
-class B(typing.List): ...
+class _Dict(typing.Dict): ...
 
 # TODO: should have `Generic`, should not have `Unknown`
-reveal_type(B.__mro__)  # revealed: tuple[Literal[B], Literal[list], Unknown, Literal[object]]
+reveal_type(_Dict.__mro__)  # revealed: tuple[Literal[_Dict], Literal[dict], Unknown, Literal[object]]
 
-class C(typing.Set): ...
+class _Set(typing.Set): ...
 
 # TODO: should have `Generic`, should not have `Unknown`
-reveal_type(C.__mro__)  # revealed: tuple[Literal[C], Literal[set], Unknown, Literal[object]]
+reveal_type(_Set.__mro__)  # revealed: tuple[Literal[_Set], Literal[set], Unknown, Literal[object]]
 
-class D(typing.FrozenSet): ...
+class _FrozenSet(typing.FrozenSet): ...
 
 # TODO: should have `Generic`, should not have `Unknown`
-reveal_type(D.__mro__)  # revealed: tuple[Literal[D], Literal[frozenset], Unknown, Literal[object]]
+reveal_type(_FrozenSet.__mro__)  # revealed: tuple[Literal[_FrozenSet], Literal[frozenset], Unknown, Literal[object]]
+
+####################
+### `collections`
+
+class _ChainMap(typing.ChainMap): ...
+
+# revealed: tuple[Literal[_ChainMap], @Todo(Support for more typing aliases as base classes), Literal[object]]
```

Comment:
Can you resolve this TODO as well, while we're here? It should be trivial, given the other changes you've already made as part of this PR

Reply:
you just need to fill in this branch of the `match` statement: https://github.com/astral-sh/ruff/blob/a623d8f7c44cd63897489cad0faed1538d4373c8/crates/red_knot_python_semantic/src/types/class_base.rs#L115-L123